runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/DoFnOperator.java

```diff
@@ -418,33 +451,7 @@ public void close() throws Exception {
   }
 
   private long getPushbackWatermarkHold() {
```

Comment:
Maybe remove the method ? since now the state is kept in the variable, just access directly.

Reply:
I kept them for now to not have too many changes but can do that as a follow up